src/mlpack/methods/softmax_regression/softmax_regression.hpp

```diff
@@ -206,6 +239,8 @@ class SoftmaxRegression
  private:
   //! Parameters after optimization.
   arma::mat parameters;
+  //! Input size
+  size_t inputSize;
```

Comment:
Maybe I missed something but it looks like we don't need a local copy for the `inputSize` parameter.